<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon.qrc">
    <normaloff>:/icon/icon.svg</normaloff>:/icon/icon.svg</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="3">
       <widget class="QPushButton" name="installButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>安装系统</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="systemChooser">
        <item>
         <property name="text">
          <string>bookworm</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>sid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>bullseye</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>buster</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="refreshDiskList">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>刷新磁盘列表</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="systemChooserTips">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>安装系统：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QTableView" name="diskChooser"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="partTips">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;如果需要添加/删除分区等操作，请使用其他分区工具操作，如 gparted&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="editChoosePart">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>编辑选中分区</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QFrame" name="userInformation">
        <property name="toolTip">
         <string/>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QLabel" name="rootPasswordTips0">
           <property name="text">
            <string>root 用户密码：&lt;a style='color: red;'&gt;&lt;b&gt;×&lt;/b&gt;&lt;/a&gt;没有填写</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rootPassword0">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="rootPasswordTips1">
           <property name="text">
            <string>再输一次root用户密码：&lt;a style='color: red;'&gt;&lt;b&gt;×&lt;/b&gt;&lt;/a&gt;没有填写</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="rootPassword1">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&lt;hr&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="userNameTips">
           <property name="text">
            <string>用户名：&lt;a style='color: red;'&gt;&lt;b&gt;×&lt;/b&gt;&lt;/a&gt;没有填写</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="userName">
           <property name="echoMode">
            <enum>QLineEdit::Normal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="userPasswordTips0">
           <property name="text">
            <string>用户密码：&lt;a style='color: red;'&gt;&lt;b&gt;×&lt;/b&gt;&lt;/a&gt;没有填写</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="userPassword0">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="userPasswordTips1">
           <property name="text">
            <string>再输一次用户密码：&lt;a style='color: red;'&gt;&lt;b&gt;×&lt;/b&gt;&lt;/a&gt;没有填写</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="userPassword1">
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>&lt;hr&gt;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="hostNameTips">
           <property name="text">
            <string>主机名：&lt;a style='color: red;'&gt;&lt;b&gt;×&lt;/b&gt;&lt;/a&gt;没有填写</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="hostName"/>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1007</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_P">
    <property name="title">
     <string>程序（&amp;P）</string>
    </property>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_H">
    <property name="title">
     <string>帮助（&amp;H）</string>
    </property>
    <addaction name="action_about"/>
    <addaction name="separator"/>
    <addaction name="action_qqGroup"/>
   </widget>
   <addaction name="menu_P"/>
   <addaction name="menu_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_exit">
   <property name="text">
    <string>退出（&amp;E）</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于（&amp;A）</string>
   </property>
  </action>
  <action name="action_qqGroup">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>QQ 交流群：881201853</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
